docs/sources/tempo/release-notes/v2-4.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Version 2.4 release notes
+menuTitle: V2.4
+description: Release notes for Grafana Tempo 2.4
+weight: 40
+---
+
+# Version 2.4 release notes
+
+The Tempo team is pleased to announce the release of Tempo 2.4.
+
+This release gives you:
+
+* New features, including multi-tenant queries and experimental TraceQL metrics queries
+* Performance enhancements, thanks to the addition of new caching tiers
+* Cost savings, thanks to polling improvements that reduce calls to object storage
+
+As part of this release, vParquet3 has also been promoted to the new default storage format for traces. For more about why we’re so excited about vParquet3, refer to [Accelerate TraceQL queries at scale with dedicated attribute columns in Grafana Tempo](/blog/2024/01/22/accelerate-traceql-queries-at-scale-with-dedicated-attribute-columns-in-grafana-tempo/).
+
+Read the [Tempo 2.4 blog post](/blog/2023/11/01/grafana-tempo-2.3-release-faster-trace-queries-traceql-upgrades/) for more examples and details about these improvements.
+
+These release notes highlight the most important features and bugfixes. For a complete list, refer to the [Tempo changelog](https://github.com/grafana/tempo/releases).
+
+## Features and enhancements
+
+The most important features and enhancements in Tempo 2.4 are highlighted below.
+
+### Multi-tenant queries
+
+Tempo now allows you to query multiple tenants at once. We’ve made multi-tenant queries compatible with streaming ([first released in v2.2]({{< relref "./v2-2#get-traceql-results-faster" >}})) so you can get query results as fast as possible. To learn more, refer to [Cross-tenant federation]({{< relref "../operations/cross_tenant_query" >}}) and [Enable multi-tenancy]({{< relref "../operations/multitenancy" >}}). [PRs [3262](https://github.com/grafana/tempo/pull/3262), [3087](https://github.com/grafana/tempo/pull/3087)]
+
+### TraceQL metrics (experimental)
+
+We’re excited to announce the addition of metrics queries to the TraceQL language. Metric queries extend trace queries by applying a function to trace query results. This powerful feature creates metrics from traces, much in the same way that LogQL metric queries create metrics from logs.
+
+Tempo 2.4 introduces metrics queries to the TraceQL language as an experimental feature.
+
+Metric queries extend trace queries by applying a function to trace query results.
+
+This powerful feature creates metrics from traces, much in the same way that LogQL metric queries create metrics from logs.
+
+Initially, only `count_over_time` and `rate` are supported.
```

Comment:
this seems repeated content from the traceql metrics page? what do you think about just teasing a rate query and then linking to those docs?

also, should we drop mentions of `count_over_time`? i just don't think it makes sense unless you provide a duration (which we don't support yet) b/c you have no idea what the time period the counts are evaluated over. 

cc @mdisibio

Reply:
I'll see if I can shorten up the section.

Reply:
So we are saying we only support rate? I am thinking about just dropping the sentence entirely and just teasing an example and linking to the doc page for TraceQL metrics.

Reply:
> I am thinking about just dropping the sentence entirely and just teasing an example and linking to the doc page for TraceQL metrics.
Sounds good! 

> So we are saying we only support rate?
If you run a query with "count_over_time" it will work and return an answer, so in that regard its "supported". The issue is that in the UI its not obvious the range of time the count is executed over. So if you mouse over a point on the line and you see the value `53`, you know the count of spans is `53`, but you don't know "was that 53 spans seen over a minute? over 30 seconds? over 10 minutes?". With rate, you know that the unit is "spans/second" so if you saw "53" you'd know it was "53 spans per second".

Reply:
thank you for the explanation. when in doubt, leave it out...